Automattic/woocommerce-payments
# Move the overview task list to the welcome greeting to improve visibility

Fixes #6382 

#### Changes proposed in this Pull Request









This PR moved the Payments → Overview `Things to do` task list to be rendered under the `Welcome` greeting to improve visibility of tasks.

* When there are tasks to be displayed, the Welcome card is split from the AccountBalances card.
* When there are no tasks to be displayed, the Welcome & AccountBalances cards are displayed as a single card.


#### Testing instructions






* View the Payments → Overview screen with no disputes.
	* The Welcome/Balances card should not be split.
	* The TaskList should not be visible.

   	![image](https://github.com/Automattic/woocommerce-payments/assets/3147296/ac9d9bd2-0990-48f8-90f0-4c88b80be108)

* View the Payments → Overview with 1+ active disputes.
	* The Welcome card should be split from the Balances card.
	* The TaskList should be visible under the Welcome card header with a dispute resolution task.
	* The `Things to do` task list should not be visible elsewhere on the Overview page.
	
   	![image](https://github.com/Automattic/woocommerce-payments/assets/3147296/f4ae1521-a8a2-48c7-bb07-0d02066ae416)

## TODO

- [x] [Confirm which tasks are moving](https://github.com/Automattic/woocommerce-payments/pull/6397#issuecomment-1577518516) / if we are splitting the `Things to do` task list into 2 lists and including only urgent tasks in the Welcome banner.
	- Confirmed: **all existing tasks should appear in the welcome banner header**, so merchants know where can find any tasks. It would be confusing to have different tasks in different places. 

-------------------


**Post merge**
